.github/workflows/pandoc.yml

```diff
@@ -0,0 +1,27 @@
+name: Create README.pdf
+
+on: push
+
+jobs:
+  publish_readme_pdf:
+    name: README as pdf
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Convert
+        uses: docker://pandoc/latex:2.14.2
+        with:
+          args: >-
+            --output=README.pdf
```

Comment:
There's a strange color choose.
<img width="613" alt="image" src="https://user-images.githubusercontent.com/19565938/139785320-d3f4fc2b-a5af-4127-ba13-e3c3432903ed.png">

Reply:
That's because of the newline so that the text doesn't go out of the page, I don't think I can fix this.